test/python/test_mapper.py

```diff
@@ -260,6 +262,13 @@ def test_move_measurements(self):
         meas_nodes = out_dag.get_named_nodes('measure')
         self.assertEqual(len(moved_meas), len(meas_nodes))
 
+    def test_kak_decomposition(self):
+        """Verify KAK decomposition for random Haar unitaries.
+        """
+        for _ in range(100):
```

Comment:
If this test raises, it won't fail, it will _error_. To make it fail and provide a meaningful reason, you can make:
```python
for _ in range(100):
  unitary = random_unitary_matrix(4)
  with self.subTest(unitary=unitary):
   try:
     two_qubit_kak(unitary, verify_gate_sequence=True):
   except MapperError as ex:
     self.fail(str(ex))
```

Reply:
Ok, cool will fix.